sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuOverrides.scala

```diff
@@ -2437,6 +2437,17 @@ object GpuOverrides {
               .withPsNote(TypeEnum.STRING, "only a single character is allowed"), TypeSig.STRING),
           ParamCheck("count", TypeSig.lit(TypeEnum.INT), TypeSig.INT))),
       (in, conf, p, r) => new SubstringIndexMeta(in, conf, p, r)),
+    expr[StringRepeat](
+      "StringRepeat operator",
+      ExprChecks.projectNotLambda(TypeSig.STRING, TypeSig.STRING,
+        Seq(ParamCheck("str", TypeSig.STRING, TypeSig.STRING),
```

Comment:
I'm not sure if this is the correct check.

The first input to this function can either be a string scalar or a strings column.